tests/test_workers/test_celery_logging.py

```diff
@@ -1,28 +1,80 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 import logging
+import os
 from unittest import mock
 
 from celery.utils.log import ColorFormatter
 
 from cachito.workers import celery_logging
 
 
-def test_setup_task_logging(caplog):
-    # Setting the logging level via caplog.set_level is not sufficient. The Flask
-    # related settings from previous tests interfere with this.
-    workers_logger = logging.getLogger("cachito.workers")
-    workers_logger.disabled = False
-    workers_logger.setLevel(logging.INFO)
+def test_cleanup_task_logging(tmpdir):
+    # Add a file handler first to test remove handler function
+    request_log_handler = logging.FileHandler("fake-path")
+    logger = logging.getLogger()
+    logger.addHandler(request_log_handler)
+
+    file_handler_count = 0
+    for handler in logger.handlers:
+        if isinstance(handler, logging.FileHandler):
+            file_handler_count = 1
+            break
+    # Assert that file handler was created
+    assert file_handler_count == 1
 
+    try:
+        celery_logging.cleanup_task_logging(mock.Mock(), mock.Mock())
+        file_handler_count = 0
+        for handler in logger.handlers:
+            if isinstance(handler, logging.FileHandler):
+                file_handler_count = 1
+                break
+        # Assert that handler was removed
+        assert file_handler_count == 0
```

Comment:
```suggestion
        for handler in logger.handlers:
            assert not isinstance(handler, logging.FileHandler)
```

Reply:
I opted for the different approach because I tried what you suggested first and it doesn't work, since it creates a root logger first and then a file handler, so there are basically 2 items in logger.handlers and only one of them is filehandler.

Reply:
@nirzari right, aren't you just checking to be sure there is no file handler on the root logger?